File: LeetCode/Easy/Richest Customer Wealth/1672.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int maximum = 0;
        for (int[] account : accounts) {
            int sum = 0;
            for (int money : account) sum += money;
            if (maximum < sum) maximum = sum;
        }
        return maximum;
    }
}

File: LeetCode/Easy/Richest Customer Wealth/Richest Customer Wealth/1672.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    let maximum = 0;
    for (const account of accounts) {
        let sum = 0;
        for (const money of account) sum += money;
        if (maximum < sum) maximum = sum;
    }
    return maximum;
};
